optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, config: LlamaConfig):
         self.past_key = None
         self.past_value = None
         self.inp_seq_len = -1
-        self.register_buffer("norm_factor", torch.tensor(1.0 / math.sqrt(self.head_dim)), persistent=False)
+        self.norm_factor = torch.tensor(1.0 / math.sqrt(self.head_dim))
```

Comment:
```suggestion
        self.norm_factor = 1.0 / math.sqrt(self.head_dim)
```
Actually it doesn't have to be a tensor as it is just a single float.